lib/mobility.rb

```diff
@@ -186,14 +185,6 @@ def set_locale(locale)
     end
   end
 
-  module ClassMethods
-    # Return translated attribute names on this model.
-    # @return [Array<String>] Attribute names
-    def mobility_attribute?(_)
```

Comment:
Previously this had to be defined here because it was used by the uniqueness validator, which was immediately included in a model with `include Mobility`, and would break unless there was a `translates` call. But now the uniqueness validator is only included when `translates` is called, so we don't need this.